projects/plugins/jetpack/_inc/client/components/agencies-card/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import Button from 'components/button';
+import Card from 'components/card';
+import analytics from 'lib/analytics';
+import { FC, useCallback } from 'react';
+import { connect } from 'react-redux';
+import { isCurrentUserLinked, isConnectionOwner, connectUser } from 'state/connection';
+import { agenciesLearnMoreDismissed, updateSettings } from 'state/settings';
+import { isFetchingSiteData } from 'state/site';
+
+interface Props {
+	isDismissed: boolean;
+	dismiss: () => void;
+	path: string;
+	isUserLinked: boolean;
+	isOwner: boolean;
+	isFetchingData: boolean;
+	className?: string;
+}
+
+const AgenciesCard: FC< Props > = ( {
+	isDismissed,
+	dismiss,
+	path,
+	isUserLinked,
+	isOwner,
+	isFetchingData,
+	className,
+} ) => {
+	const trackEvent = useCallback(
+		( target: string ) => {
+			analytics.tracks.recordJetpackClick( {
+				target: target,
+				feature: 'agencies',
+				page: path,
+				is_user_wpcom_connected: isUserLinked ? 'yes' : 'no',
+				is_connection_owner: isOwner ? 'yes' : 'no',
+			} );
+		},
+		[ path, isUserLinked, isOwner ]
+	);
+
+	const handleDismiss = useCallback( () => {
+		dismiss();
+		trackEvent( 'learn-more-dismiss' );
+	}, [ dismiss, trackEvent ] );
+
+	if ( isDismissed ) {
+		return null;
+	} else if ( isFetchingData ) {
+		return <div />;
```

Comment:
We might want to show a placeholder here. Since the component is fairly far down the page, this would not provide much value at the moment though.

Reply:
The module below currently uses this syntax, and as you said it's pretty low on the page so in this case I think I'll stick to being consistent with similar modules 😄